### PR TITLE
Show correct motion team in decision dialog

### LINKED ISSUE #3842: Team picker is displaying incorrect team
## Steps to reproduce

1. Create a new team. But don't select it, i.e have "All teams" selected.
2. Click "New Decision". 
3. It shows the team you've just created in the domain selector.

## Expected behaviour

1. Team should be the team that is selected. 

## Actual behaviour

1. Team is not the one that is selected. 

teams

### PR DESCRIPTION
**Pull request by chinins**
## Description

Updates decisions page wrapper and decisions dialog to pass correct props.

resolves #3842

### CODE REVIEW

File: src/modules/dashboard/components/ColonyDecisions/ColonyDecisionsWrapper.tsx [hunk 1]
```diff
@@ -5,13 +5,20 @@ import ColonyNewDecision from '~dashboard/ColonyNewDecision';
...
+  filteredDomainId: number;
+  onDomainChange?: (domainId: number) => void;
+}
+
 const ColonyDecisionsWrapper = ({
   colony,
   filteredDomainId,
-  setDomainIdFilter,
+  onDomainChange,
 }: any) => {
```
Comment (jakubcolony) [inline src/modules/dashboard/components/ColonyDecisions/ColonyDecisionsWrapper.tsx:22]: Don't you want to use the newly defined `Props` here?

**Review by jakubcolony — COMMENTED** [review]
Looks good to me! I wasn't able to reproduce the bug.

File: src/modules/dashboard/components/ColonyDecisions/ColonyDecisionsWrapper.tsx (hunk 1 above)
Comment (chinins) [reply src/modules/dashboard/components/ColonyDecisions/ColonyDecisionsWrapper.tsx:22]: of course, that was the creation's purpose  🙈

**Review by danbr — APPROVED** [review]
Looks good - functions as expected.

Please note - <outside the scope of this PR> I noticed a bug later, the Team displayed is incorrect when staking. If an issue/PR doesnt already exist then i will create.

**Comment by chinins** [comment]
> Looks good - functions as expected.
> 
> Please note - I noticed a bug later, the Team displayed is incorrect when staking. If an issue/PR doesnt already exist then i will create.

I have already created an issue for that and am working on it. It seems that it's due the decision saga but still investigating.

**Pull request merged**